gym_electric_motor/physical_systems/electric_motors.py

```diff
@@ -1918,9 +1918,6 @@ def _update_limits(self, limit_values={}, nominal_values={}):
                                self._limits[u] / self._motor_parameter['r_s']
             nominal_agenda[i] = self._nominal_values.get('i', None) or \
                                 self._nominal_values[u] / self._motor_parameter['r_s']
-        # overwrite default limits and nominal values with func args
-        limits_agenda.update(limit_values)
-        nominal_agenda.update(nominal_values)

```

Comment:
I think these lines simply revert the changes made to limits_agenda, so the limits stay the same with these lines.